test/functional/specs/desktop/C2583.js

```diff
@@ -0,0 +1,87 @@
+import { t, Selector } from "testcafe";
+
+const urlCollector = `http://127.0.0.1:8080/test/functional/sandbox/html/alloySdk.html`;
+
+fixture`C2583`.page(urlCollector);
+
+test.meta({
+  ID: "C2583",
+  SEVERITY: "P0",
+  TEST_RUN: "Regression"
+});
+
+test("Regression: Set the log option to true. Load the page. Execute an event command.", async () => {
+  await t
+    .click(
+      Selector(
+        "#body > section > div.left-nav > div > ul > li:nth-child(1) > a"
```

Comment:
Rather than having these complex selectors that might break if you restructure the HTML, I recommend adding a data attribute to each of the links and then selecting off the data attribute.

https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes
https://css-tricks.com/almanac/selectors/a/attribute/